api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,27 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
```

Comment:
It looks like that is only true of persistent actors because that logic is contained in `HmdaPersistentActor`, but I think that this might still be a problem for the `InstitutionPersistence` clause.